units/src/amount.rs

```diff
@@ -865,9 +865,9 @@ impl Amount {
     pub fn checked_rem(self, rhs: u64) -> Option<Amount> { self.0.checked_rem(rhs).map(Amount) }
 
     /// Convert to a signed amount.
-    pub fn to_signed(self) -> Result<SignedAmount, ParseAmountError> {
+    pub fn to_signed(self) -> Result<SignedAmount, OutOfRangeError> {
```

Comment:
I meant to make this change a separate commit but don't worry about it, it's not important.